src/components/Accordion/Accordion.stories.tsx

```diff
@@ -93,6 +93,16 @@ export const WithDifferentAccordionItems: Story<AccordionProps> = () => {
                 </div>
                 <Button onClick={() => setShowContent(!showContent)}>Toggle Content</Button>
             </AccordionItem>
+            <AccordionItem
+                header={{ children: "Item without padding", type: FieldsetHeaderType.AddRemove }}
+                noPadding={true}
```

Comment:
```suggestion
                padding={false}
```
We usually put "positive" props

Reply:
Changed :)